src/modules/roles/application/use-cases/__tests__/update-user-roles.use-case.spec.ts

```
@@ -0,0 +1,367 @@
+        id: userId,
+        roles: [existingRole],
+      };
+
+      userRepo.findOneOrFail.mockResolvedValue(user);
+      roleRepo.findOneOrFail.mockResolvedValue(newRole);
+      userRepo.save.mockResolvedValue({
+        ...user,
+        roles: [existingRole, newRole],
+      });
+
+      const result = await useCase.execute(userId, roleId, undefined, currentUserId);
```

ESLint

[error] 'result' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)